lib/python/treadmill_aws/plugins/profile.py

```diff
@@ -1,4 +1,19 @@
-"""AWS Profile."""
+"""AWS profile plugin.
+"""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import os
+
+
+def api_scope():
+    """Returns admin API DNS scope."""
+    return [os.environ.get('TREADMILL_CELL', 'na') + '.' + 'cell']
```

Comment:
no, it is more involved. what we need to change is not to put admin api dns records into cell zone, rather manage it in global zone.